package.json

```diff
@@ -18,6 +18,7 @@
     "ocsp": "^1.2.0",
     "request": "^2.88.0",
     "simple-lru-cache": "^0.0.2",
+    "snowflake-sdk": "^1.5.0",
```

Comment:
Please remove this. 
This package.json is to install the dependencies and not the NodeJS connector itself.